includes/RestApi/SettingsController.php

```diff
@@ -202,18 +203,20 @@ public function update_item( \WP_REST_Request $request ) {
 	 */
 	public function get_current_settings() {
 
+		$social_data = \get_option( $this->yoast_wp_options_key );
```

Comment:
Can we move `yoast_wp_options_key` to the `Options` class? and also return a default value incase the option does not exist?